ansible/roles/opencontrail-controller/handlers/main.yml

```diff
@@ -0,0 +1,6 @@
+---
+- name: restart contrail-control
+  command: docker kill $(docker ps | awk '/opencontrail\/control/ {print $1;}')
```

Comment:
This handler assumes Docker is installed, but the role `opencontrail-controller` only depends on `kubernetes` so you can not assume Docker is installed.